data/mods/Aftershock/items/crafting_scrap/heat_transfer_scrap.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "type": "GENERIC",
+    "id": "afs_heat_1",
+    "copy-from": "afs_scrap_1",
+    "name": { "str": "conductive thermoplastic" },
+    "description": "An excellent heat conductor but also an electrical insulator, conductive thermoplastics were introduced to the civilian market nearly two centuries ago, and have tightly integrated itself into the modern society since then.",
```

Comment:
```suggestion
    "description": "An excellent heat conductor but also an electrical insulator, conductive thermoplastics were introduced to the civilian market nearly two centuries ago and have tightly integrated themselves into the modern society since then.",
```